src/utils.js

```diff
@@ -66,28 +66,8 @@ export function getDefaultRegistry() {
   };
 }
 
-export function defaultTypeValue(schema) {
-  if (typeof schema.default !== "undefined") {
-    return schema.default;
-  }
-  const {type} = schema;
-  switch (type) {
-  case "array":     return [];
-  case "boolean":   return false;
-  case "number":    return 0;
-  case "object":    return {};
-  case "string":    {
-    if (schema.format === "date-time") {
-      return new Date().toJSON();
```

Comment:
That particular one is a breaking change, no ?

How do we get date-time fields to default to current date ? It should take the current datetime, not the one when the app/schema was instantiated.